thirdeye-ui/src/app/utils/time-range/time-range.util.ts

```diff
@@ -199,28 +199,32 @@ export const formatTimeRange = (timeRange: TimeRange): string => {
 
 // Returns formatted string representation of time range duration
 export const formatTimeRangeDuration = (
-    timeRangeDuration: TimeRangeDuration
+    timeRangeDuration: TimeRangeDuration,
+    timezone?: string
```

Comment:
Do you think having this default to `"UTC"` would be better or not?

Reply:
I'm leaving the option open to view in the local timezone by having this be undefined